build.sh

```diff
@@ -5,13 +5,12 @@ TAG=$(date +%Y-%m-%d)
 REPO=$1
 VERSION=$2
 GCLOUD_CMD="gcloud"
-if [ -n $3 ]
+if [ "$3" != "" ]
```

Comment:
a little nitpicky, but I believe this is the same as `[ -z "$3" ]`, which seems a little cleaner to me.